providers/src/airflow/providers/edge/models/edge_job.py

```diff
@@ -59,6 +59,7 @@ class EdgeJobModel(Base, LoggingMixin):
     try_number = Column(Integer, primary_key=True, default=0)
     state = Column(String(20))
     queue = Column(String(256))
+    need_capacity = Column(Integer, default=1)
```

Comment:
As you require the field to be set, you have no migration as you drop the table... no default is needed.
```suggestion
    need_capacity = Column(Integer)
```

Reply:
done